.scrutinizer.yml

```diff
@@ -20,9 +20,11 @@ build:
     dependencies:
         before:
             - "yes '' | pecl -q install -f mongo-stable"
+            - "yes '' | pecl -q install -f mongodb-stable"
             - "sudo composer self-update"
         override:
-            - "composer install --no-dev --no-interaction --prefer-dist"
+            - composer update --no-interaction --prefer-dist
```

Comment:
If you use [external code coverage](https://scrutinizer-ci.com/docs/tools/external-code-coverage/) no composer is needed here.